app/templates/public/sessions.hbs

```diff
@@ -53,6 +53,8 @@
 
 <div class="mt-8">
   {{#if this.model.session}}
+  {{#each this.groupByDateSessions as |group|}}
+    <h3 class="ui header {{if this.device.isMobile 'mb--8' 'mb--4'}}">{{t group.date}}</h3>
```

Comment:
t doesn't work with dynamic data, and secondly, dates are not translatable, as mentioned previously